packages/plugin-ext/src/plugin/tree/tree-views.ts

```diff
@@ -88,6 +89,22 @@ export class TreeViewsExtImpl implements TreeViewsExt {
             get onDidChangeVisibility() {
                 return treeView.onDidChangeVisibility;
             },
+            // tslint:disable-next-line:typedef
+            get message() {
+                return treeView.message;
+            },
+            // tslint:disable-next-line:typedef
```

Comment:
i don't believe set can typedef, btw we don't use tslint, but eslint

Reply:
This was added before we switched to eslint. I'm about to remove these typedef-s, but then should I add return type (string) to be `get message(): string {` and `get title(): string {`? 
Other existing getters do not specify return type...

Reply:
(with tslint without typedef added for `set` it was resulting into compilation errors. A bit strange)